core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -91,4 +91,6 @@ private TableProperties() {}
 
   public static final String DEFAULT_WRITE_METRICS_MODE = "write.metadata.metrics.default";
   public static final String DEFAULT_WRITE_METRICS_MODE_DEFAULT = "truncate(16)";
+
+  public static final String DEFAULT_NAME_MAPPING = "schema.name-mapping.default";
```

Comment:
We're calling this default, but it's not really a default, right?  Shouldn't this just be `schema.name-mapping`?

Reply:
The idea is that there may eventually be multiple mappings. If you have two streams of data written by Kafka, for example, you may want a different mapping from source name to Iceberg columns.

The current name defines a default mapping, to be used when a mapping is needed but there is no specified mapping for a file. Later, we can add `schema.name-mapping.(name)` properties to add more than one.